setup.py

```diff
@@ -1,44 +1,28 @@
-import distutils
 import os
 import shutil
 import subprocess
 import sys
 
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext as build_ext_orig
 
 import versioneer
 
 
-class MavenCommand(distutils.cmd.Command):
-    """Run the maven build command"""
-
-    description = "run the mvn install command"
-    user_options = []
-
-    def initialize_options(self):
-        """No options"""
-        pass
-
-    def finalize_options(self):
-        """No options"""
-        pass
```

Comment:
If we want to have an option to enable/disable here, we could leverage these to provide that for us. No strong opinion on whether or not we do this

Reply:
Yeah I think something like that would be nice if we ever want to install the Python bindings alone